attachments/views.py

```diff
@@ -1,21 +1,26 @@
+import json
+
 from django.shortcuts import render_to_response, get_object_or_404
 from django.http import HttpResponseRedirect, Http404, HttpResponse
 from django.template import RequestContext
 from django.contrib.auth.decorators import login_required
 from django.contrib.contenttypes.models import ContentType
 from django.core import serializers
-from django.utils import simplejson
 
 from attachments.models import Attachment
 from attachments.forms import AttachmentForm, AttachmentEditForm
 
 
 @login_required
-def new_attachment(request, content_type, object_id,
-                   template_name='attachments/new_attachment.html',
-                   form_cls=AttachmentForm,
-                   redirect=lambda object, attachment: object.get_absolute_url()):
-    object_type = get_object_or_404(ContentType, id = int(content_type))
+def new_attachment(
+    request,
+    content_type,
+    object_id,
+    template_name='attachments/new_attachment.html',
+    form_cls=AttachmentForm,
+    redirect=lambda object, attachment: object.get_absolute_url(),
+):
+    object_type = get_object_or_404(ContentType, id=int(content_type))
     try:
         object = object_type.get_object_for_this_type(pk=int(object_id))
```

Comment:
Yea, it's not good. It's actually pretty concerning that attachments are such a huge part of PS and the code is so bad.